sdk/core/src/entities/packages/package.rs

```diff
@@ -124,7 +124,12 @@ impl Package {
             vulnerabilities: vec![],
         };
 
-        package.join_task(purl, task)?;
+        match task {
```

Comment:
Clippy is going to complain about this.  It wants you to use the: `if let Some(task) = task {}` style when you don't have a branch for `None`

Reply:
seems to pass as is